public/api-docs/swaggerDoc.json

```diff
@@ -177,9 +177,129 @@
+                "description": "",
+                "operationId": "loginUser",
+                "produces": [
+                    "application/json"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "successful operation",
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "url":{
+                                "type": "string",
+                                "example": "https://www.linkedin.com/uas/oauth2/authorization?response_type=code&client_id=86h0yv5ea8q7ti"
+                                }
+                           }
+                        }
+                    },
+                    "500": {
+                        "description": "error message"
+                    }
+                }
+            }
+        },
+        "/api/v1/auth/google/callback": {
+            "get": {
+                "tags": [
+                    "User"
+                ],
+                "summary": "redirect route for Google to Log user into the system",
+                "description": "",
+                "operationId": "loginUser",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "in": "body",
+                        "name": "signin",
+                        "schema": {}
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "successful operation",
+                        "schema": {
+                            "$ref": "#/definitions/User"
+                        }
+                    },
+                    "404": {
+                        "description": "User not found"
+                    }
+                }
+            }
+        },
+        "/api/v1/auth/linkedin/callback": {
+            "get": {
+                "tags": [
+                    "User"
+                ],
+                "summary": "Linkedin redirect route to Log user into the system",
+                "description": "",
+                "operationId": "loginUser",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "in": "body",
+                        "name": "signin",
+                        "schema": {}
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "successful operation",
+                        "schema": {
+                            "$ref": "#/definitions/User"
+                        }
                     },
-                    "422": {
-                        "description": "Invalid Input"
```

Comment:
Why did you remove this?

Reply:
I actually replaced it with ```  "404": {
                        "description": "User not found"```. Since it doesn't describe the error response of the specified endpoint.

Reply:
It does describe errors from the input, mostly errors from express-validator

Reply:
@okobaba1 Thanks for pointing that out. However, the 422 error code wasn't specified in any of my enpoints, instead, I used 404 as appropriate. Since I'm only using a GET request I believe there are no input parameters to validate.

Reply:
I think you removed it from the signup.

Reply:
No. This is what is currently present in signup ``` "409": {
                        "description": "User already exists"
                    }```

Reply:
409 and 422 error codes were both present in signup, please add the 422 description you removed